packages/harvester-cloud-provider/package.yaml

```diff
@@ -0,0 +1,3 @@
+url: https://github.com/harvester/charts.git
+subdirectory: charts/harvester-cloud-provider
+commit: c7746e5bd688d41ca7b984e545166f887e9eefc2
```

Comment:
Need to add version:
https://github.com/rancher/charts/blob/33db4185684ece7b2ca1575cdc66af4841967764/packages/longhorn/package.yaml#L4

Reply:
No need to change the Chart.yaml; scripts will take care of that automatically. Just need to add this flag to package.yaml